azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -345,6 +347,7 @@ def _format_microsoft_v2_results(self) -> List[Dict[str, Any]]:
         
         status = self.status()
 
+        # TODO: construction of header is quite minimal compared to passthrough flow - evaluate if we want to add more information

```

Comment:
What kind of information were you thinking about?

Reply:
I'm not sure - not familiar with what headers are used for really, would be good to discuss

Reply:
Could you please investigate how is this usually populated in Qiskit?